packages/tenant-process/src/services/tenantService.ts

```diff
@@ -363,6 +366,81 @@ export function tenantServiceBuilder(
       return updatedTenant;
     },
 
+    async revokeCertifiedAttributeById(
+      {
+        tenantId,
+        attributeId,
+        organizationId,
+        correlationId,
+      }: {
+        tenantId: TenantId;
+        attributeId: AttributeId;
+        organizationId: TenantId;
+        correlationId: string;
+      },
+      logger: Logger
+    ): Promise<void> {
+      logger.info(
+        `Revoke certified attribute ${attributeId} to tenantId ${tenantId}`
+      );
+      const requesterTenant = await retrieveTenant(
```

Comment:
to be precise, this should return 500 but it is handled with a 404

Reply:
I suppose this comes from scala version, but I agree this should handled as 404.
May I suggest to fix after migration?